src/controllers/likeController.js

```diff
@@ -19,7 +19,22 @@ const unlikeFeeds = catchAsync(async (req, res) => {
   res.status(200).json(like);
 });
 
+const getLike = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const { feedId } = req.params;
+
+  const like = await likeService.getLike(userId, feedId);
+
+  if (like) {
+    res.status(200).json({ liked: true, like });
+  } else {
+    res.status(200).json({ liked: false });
+  }
+});
```

Comment:
좋아요 목록이 없으면 없는대로 빈 배열을 반환해 주시면 됩니다.
```suggestion

    res.status(200).json({ liked: true, like });
});
```